config/sync/core.extension.yml

```diff
@@ -17,6 +17,7 @@ module:
   config: 0
   config_ignore: 0
   config_import_locale: 0
+  content_fixtures: 0
```

Comment:
The installation of this (and other fixture) modules should not be committed to git.

Run

```shell
docker compose exec phpfpm vendor/bin/drush --yes pm:uninstall content_fixtures
```

to clean up (cf. https://github.com/itk-dev/hoeringsportal/blob/develop/documentation/localDevelopment.md).